app/views/import_vat/import_vat.scala.html

```diff
@@ -83,9 +83,13 @@ <h2 class="govuk-heading-s" id="historic-eori-@{historyIndex}">@messages("cf.acc
 
     @missing_guidance("certificate")
 
-    @p(id = Some("historic-statement-request"),
-        classes = "govuk-body govuk-!-padding-bottom-7",
-        message = "cf.account.vat.historic.description",
-        link = Some(link(msg = "cf.account.vat.historic.request", url = appConfig.historicRequestUrl(C79Certificate))))
+        <div class="govuk-body govuk-!-padding-bottom-7">
```

Comment:
To align with the recent changes to using components in the views should we be using a P component with the link set to none & then we can use the link component for the link removed from the P.